### PR TITLE
Adding support of Export, Cmk and BYOK features with version upgrade.

### PR DESCRIPTION
**Pull request by dhawal777**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed. This is a requirement before review can be continued.
Also please resolve the failures identified by the automated checks.

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json [hunk 1]
```diff
@@ -0,0 +1,4685 @@
...
+                "description": "Storage Accounts ."
+              },
+              {
+                "value": "ManagedDisk",
+                "description": "Azure Managed disk storage."
+              }
+            ]
+          }
+        },
+        "sharePassword": {
```
Comment (pilor) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json:1610]: Mark this as x-ms-secret and put the appropriate x-ms-mutability on it to indicate it won't be returned in responses

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json [hunk 2]
```diff
@@ -0,0 +1,4685 @@
...
+          "readOnly": true
+        },
+        "meterType": {
+          "description": "The type of the meter.",
+          "type": "string",
+          "readOnly": true
+        },
+        "multiplier": {
+          "format": "double",
+          "description": "Multiplier specifies the region specific value to be multiplied with 1$ guid. Eg: Our new regions will be using 1$ shipping guid with appropriate multiplier specific to region.",
```
Comment (pilor) [inline specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json:3876]: users know what this means?

**Review by pilor — CHANGES_REQUESTED** [review]
The diff is very hard to review because APIs were moved around in the file.  Please avoid structural changes while making API changes unless necessary, those changes can be made in separate PRs (or a final iteration)

File: specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json (hunk 2 above)
Comment (dhawal777) [reply specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json:3880]: > users know what this means?

Yes the main consumer of this is databox portal team.

**Review by pilor — APPROVED** [review]

**Comment by dhawal777** [comment]
If we are good to merge, Please merge.

**Pull request merged**